src/plopp/core/node.py

```diff
@@ -24,18 +49,35 @@ class Node:
     """
 
     def __init__(self, func: Callable, *parents, **kwparents):
```

Comment:
The type hint doesn't match. `func` can be non-callable. It should just be `Any` now.